### PR TITLE
PHP: update required minimal php version to 7.0

### PR DESCRIPTION
**Pull request by Alexander1000**
@stanley-cheung

### CODE REVIEW

**Comment by stanley-cheung** [comment]
Can we keep it at PHP 7.0+ rather than 7.2+? The library will work with PHP 7.0+ and if a customer is at that version we don't want to exclude them.

**Comment by Alexander1000** [comment]
Ok, done!

File: templates/composer.json.template [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "homepage": "https://grpc.io",
     "license": "Apache-2.0",
     "require": {
-      "php": ">=5.5.0"
+      "php": ">=7.0.0"
```
Comment (stanley-cheung) [inline templates/composer.json.template:11]: So you will need to run the `tools/buildgen/generate_projects.sh` script to make sure these changes in the `templates/` directory are reflected in the _real_ file, in this case, the `composer.json` file at the root directory. Both changes (the file under the `templates/` directory and the real file) need to be checked in together.

File: templates/package.xml.template [hunk 2]
```diff
@@ -52,7 +52,7 @@
    <dependencies>
     <required>
      <php>
-      <min>5.5.0</min>
+      <min>7.0.0</min>
```
Comment (stanley-cheung) [inline templates/package.xml.template:55]: Same here. Running `generate_projects.sh` will cause the real file `package.xml` to be updated. Both changes need to be checked in.

File: templates/composer.json.template (hunk 1 above)
Comment (Alexander1000) [reply templates/composer.json.template:11]: thanks

File: tools/doxygen/Doxyfile.php [hunk 3]
```diff
@@ -817,7 +817,90 @@
 src/php/lib/Grpc/ServerCredentials.php \
 src/php/lib/Grpc/ServerStreamingCall.php \
 src/php/lib/Grpc/Timeval.php \
-src/php/lib/Grpc/UnaryCall.php
+src/php/lib/Grpc/UnaryCall.php \
+src/php/vendor/doctrine/instantiator/CONTRIBUTING.md \
```
Comment (stanley-cheung) [inline tools/doxygen/Doxyfile.php:821]: You probably need to clean up your composer temporary directory before running `generate_projects.sh`.

**Review by Alexander1000 — COMMENTED** [review]
thanks, generated it

File: tools/doxygen/Doxyfile.php (hunk 3 above)
Comment (Alexander1000) [reply tools/doxygen/Doxyfile.php]: ok, had no experience with doxyfiles...

**Review by stanley-cheung — APPROVED** [review]
All tests passed. Thanks for the contributions.

**Pull request merged**